README.md

```diff
@@ -1,10 +1,132 @@
-### sneaker-portfolio
-- `docker-compose up`
+# Sneaker Portfolio  
+<img src = readme-media/gifs/portfolio.gif > </img>
 
-### sneaker-server
-- `poetry run uvicorn api.main:app` to run Backend
 
-### sneaker-fronted
-- `npm install`
-- `npm run build` for building static website for FastAPI
-- `npm start`
+  
+
+# [Check out the Website here!](https://nsli.me)
+
+
+- [Sneaker Portfolio](#sneaker-portfolio)
+- [Check out the Website here!](#check-out-the-website-here)
+  - [Features](#features)
+  - [How to use?](#how-to-use)
+    - [Backend](#backend)
+    - [Frontend](#frontend)
+    - [Docker 🐳](#docker-)
+  - [API Endpoints](#api-endpoints)
+  - [Future features](#future-features)
+
+
+## Features 
+
+
+
+
+*[CLICK TO EXPAND]*
+<details>
+    <summary> 360 Degree Animations</summary>
+    <br> 
+    <img src = readme-media/gifs/360.gif > </img>
+</details>
+
+<details>
+    <summary>Drag & Drop </summary>
+    <br> 
+    <img src = readme-media/gifs/draganddrop.gif > </img>
+</details>
+
+<details>
+    <summary>Get Shoe Information & Links</summary>
+    <br> 
+    <img src = readme-media/gifs/information.gif > </img>
+</details>
+
+<details>
+    <summary>Add Shoes</summary>
+    <br> 
+    <img src = readme-media/gifs/add.gif > </img>
+</details>
+
+
+<details>
+    <summary>Remove Shoes</summary>
+    <br> 
+    <img src = readme-media/gifs/remove.gif > </img>
+</details>
+
+<details>
+    <summary>Get Random Shoe 🎲</summary>
+    <br> 
+    <img src = readme-media/gifs/random.gif > </img>
+</details>
+
+<details>
+    <summary>Favorite View ❤️</summary>
+    <br> 
+    <img src = readme-media/gifs/favorite.gif > </img>
+</details>
+
+
+
+
+
+
+## How to use?
+
+The portfolio consists of a FastAPI Backend and a Frontend written in React. For the application to be working, both of the services have to be running. 
```

Comment:
This probably shouldnt be in the How to use section

Reply:
And what is "the portfolio"

Reply:
My intention was to give an overview on why two processes have to be started in order for the application to run. However I agree that especially going into detail on fastAPI is unnecessary. I say we either leave this out completely or we write something like:
> The application consists of a back- and frontend. For it to work, both of the services have to be running.

Reply:
I commited the above text for now. If you still think this doesnt belong in this chapter, we can remove it.

Reply:
But we don't need both, if frontend is compiler we can simply use only the server.

Reply:
Ah, that is totally true. I will remove the line.